lib/performance_platform/presenter/unique_users.rb

```diff
@@ -34,5 +34,9 @@ def encode_id
     )
   end
 
+  def count_field_name
+    stats[:period] == 'month' ? 'month_count' : 'count'
+  end
+
```

Comment:
And that's exactly the presenting logic for which we have presenters, and not gateways doing everything 🐱